packages/adapters/ethers/src/tests/client.test.ts

```diff
@@ -584,7 +580,7 @@ describe('EthersAdapter', () => {
       const chainChangedHandler = mockProvider.on.mock.calls.find(
         (call: string[]) => call[0] === 'chainChanged'
       )[1]
-      await chainChangedHandler('0x1')
+      await chainChangedHandler('0x137')
```

Comment:
This will be trigger the `setCaipNetwork` only if the network has changed. So passing Polygon id now